src/components/CoinsSelectAutocomplete/CoinsSelectAutocomplete.tsx

```diff
@@ -1,411 +1,22 @@
-import { useParams } from 'react-router-dom';
-import { safeEnv } from '~/helpers';
-import { BackToHomeHelperText } from '../BackToHomeHelperText';
+import { CoinMetadata } from '~/helpers/config';
 import { CryptocurrencyIcon } from '../CryptocurrencyIcon';
 import { SelectAutocomplete } from '../SelectAutocomplete';
 import { SelectAutocompleteItem } from '../SelectAutocomplete/SelectAutocompleteItem';
 
-const buildUnsignedSweepCoins = {
```

Comment:
These have all moved to `src/helpers/config.ts` in a format with less code duplication.